tapscript/send.go

```diff
@@ -648,9 +648,14 @@ func SignVirtualTransaction(vPkt *tappsbt.VPacket, signer Signer,
 		prevAssets[input.PrevID] = input.Asset()
 	}
 
+	assetOutputs := make([]*asset.Asset, len(outputs))
+	for idx := range outputs {
+		assetOutputs[idx] = outputs[idx].Asset
+	}
+
 	// Create a Taproot Asset virtual transaction representing the asset
 	// transfer.
-	virtualTx, _, err := VirtualTx(newAsset, prevAssets)
+	virtualTx, _, err := VirtualTx(newAsset, prevAssets, assetOutputs)
```

Comment:
See above re asset outputs (they're already enumerated via the split commitment root).